### PR TITLE
House & User Settings pages

### PR DESCRIPTION
**Pull request by JSiu**
Added house settings and user settings, and associated CSS.  Removed non-required fields from registration and create house pages.

So... I'm having a fight with my VM.  Which means that I couldn't test this, and straight up coded it blind.  Going to try to fix the VM tonight, but in the meantime, I thought I'd get this in.

Backend: My understanding from the Django documentation is that if we want to populate a form with initial values (i.e., have the email field on User Settings showing their email when they load the page), that has to happen on the backend.  Let me know if that's incorrect, and if I have to do something else on the frontend.  We probably want all of the fields on both pages pre-populated, except for the password fields.

### CODE REVIEW

**Comment by creisman** [comment]
When you create the form, do SomeForm(initial={'field_name': value}) to prepopulate the field.

**Comment by JSiu** [comment]
Right... but backend was creating the forms, right? I mean, I'm sure I could figure it out, but I didn't know how the backend code was organized, so I thought I'd ask rather than poking around and possibly messing something up, especially since I can't test my code at the moment.

**Comment by j3ffml** [comment]
Actually, this should wait until the email auth change. I took all the fields out of the registration form except email and passwords1/2. If there are fields you want to prepopulate let me know, and I can do that.

**Comment by JSiu** [comment]
This shouldn't conflict with your change, Jeff; incidentally, I also took those fields out of the registration page, since we said we were moving them to the settings pages.  And all fields on both settings pages, except password fields, should be prepopulated.

**Comment by j3ffml** [comment]
Yeah, you're right this won't conflict. It just occurred to me that no one was assigned to write views for the settings pages, but I can do that now/shortly-after-this-is-merged.

File: fullhouse/templates/base.html [hunk 1]
```diff
@@ -32,8 +32,8 @@
     </a>
     {% if user.is_authenticated %}
       <ul class="nav">
-        <li><a href="#">User Settings</a></li>
-        <li><a href="#">House Settings</a></li>
+        <li><a href="/accounts/user_settings/">User Settings</a></li>
+        <li><a href="/accounts/house_settings/">House Settings</a></li>
         <li><a href="/accounts/logout/">Logout</a></li>
```
Comment (j3ffml) [inline fullhouse/templates/base.html:37]: '/acccounts/' links are for authentication views handled by auth. We probably just want these links to be `user_settings` and `house_settings` respectively, so they link to views (which I will create) in the dashboard app.

File: fullhouse/templates/user_settings.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        {% csrf_token %}
+
+        <div class="fieldWrapper">
+          {{ form.email.errors }}
+          <label form="email">Email Address: </label>
+          {{ form.email }}
+        </div>
+
+        <div class="fieldWrapper">
+          {{ form.oldpassword.errors }}
```
Comment (j3ffml) [inline fullhouse/templates/user_settings.html]: We already have a view/template for changing the password. I would suggest we leave that as its own thing; we can put a link to it here if we want.

Comment (JSiu) [inline fullhouse/templates/user_settings.html]: That link should be "/password/change/"?

Comment (j3ffml) [inline fullhouse/templates/user_settings.html]: '/accounts/password/change/'

**Comment by j3ffml** [comment]
lgtm. I'll merge this now and get the views for the settings pages working.

**Pull request merged**